api/job.go

```diff
@@ -63,20 +69,64 @@ func (c *ApiClient) FinishJob(job Job, status string) {
 	return
 }
 
-func (c *ApiClient) SendLogs(job Job, logStr string) {
-	req, err := http.NewRequest("POST", fmt.Sprintf("%s/jobs/%s/logs", c.server, job.ID), strings.NewReader(logStr))
+func (c *ApiClient) CreateStep(jobId string, name string, sortOrder int) JobStep {
```

Comment:
This is pretty crappy API code. A proper client generated from the OpenAPI spec is planned soon!